microsoft-edge/webview2/reference/javascript/webview.yml

```
@@ -0,0 +1,199 @@
+      event receives shared buffers posted from the WebView2 host via `CoreWebView2.PostSharedBufferToScript`<!-- -->.
+    remarks: ''
+    isPreview: false
+    isDeprecated: false
+    syntax:
+      content: >-
+        addEventListener<K extends keyof WebViewEventMap>(type: K, listener: (this: WebView, ev: WebViewEventMap[K]) =>
+        any, options?: boolean | AddEventListenerOptions): void;
+      parameters:
+        - id: type
+          description: >-
+            The name of the event to subscribe to. Valid values are `message`<!-- -->, and `sharedbufferreceived`<!--
+            -->.
+          type: K
+        - id: listener
+          description: ''
+          type: >-
+            (this: <xref uid="WebView2Script!WebView:class" />, ev: <xref uid="WebView2Script!WebViewEventMap:class"
+            />[K]) =&gt; any
+        - id: options
+          description: ''
+          type: boolean | AddEventListenerOptions
+      return:
+        type: void
+        description: ''
+  - name: 'addEventListener(type, listener, options)'
+    uid: 'WebView2Script!WebView#addEventListener:member(2)'
+    package: WebView2Script!
+    fullName: 'addEventListener(type, listener, options)'
+    summary: ''
+    remarks: ''
+    isPreview: false
+    isDeprecated: false
+    syntax:
+      content: >-
+        addEventListener(type: string, listener: EventListenerOrEventListenerObject, options?: boolean |
+        AddEventListenerOptions): void;
+      parameters:
+        - id: type
+          description: ''
+          type: string
+        - id: listener
+          description: ''
+          type: EventListenerOrEventListenerObject
+        - id: options
+          description: ''
+          type: boolean | AddEventListenerOptions
+      return:
+        type: void
+        description: ''
+  - name: postMessage(message)
+    uid: 'WebView2Script!WebView#postMessage:member(1)'
+    package: WebView2Script!
+    fullName: postMessage(message)
+    summary: >-
+      When the page calls `postMessage`<!-- -->, the `message` parameter is converted to JSON and is posted
+      asynchronously to the WebView2 host process. This will result in either the `CoreWebView2.WebMessageReceived`
+      event or the `CoreWebView2Frame.WebMessageReceived` event being raised depending on if `postMessage` is called
+      from the top level document in the WebView2 or a child frame.
+    remarks: |-
+        #### Examples
+
+        Post a message to the CoreWebView2:
+
+        ```TypeScript
+        const inTopLevelFrame = (window === window.parent);
+        if (inTopLevelFrame) {
+            // The message can be any JSON serializable object.
+            window.chrome.webview.postMessage({
+                myMessage: 'Hello from the script!',
+                otherValue: 1}
+            );
+            // A simple string is an example of a JSON serializable object

```

```suggestion
            // A simple string is an example of a JSON serializable object.
```
for consistency w/ other line